src/content/1.7/modules/concepts/services/_index.md

```diff
@@ -154,6 +161,82 @@ If you need more details about dependency injection and how services work in the
 their documentation about the [Service Container](https://symfony.com/doc/3.4/service_container.html).
 {{% /notice %}}
 
+### Override an existing Symfony service
+
+The container definition can be modified by a module, which enables you to override an existing Symfony service being used in PrestaShop.
+
+This is a mechanism similar to PrestaShop standard overrides, but the main benefit is that the php code stays unmodified. This prevents issues linked to code definition or autoloading failures.
+
+As you can read it from the [Symfony documentation](https://symfony.com/doc/current/service_container/service_decoration.html), there are 2 ways to modify an existing service:
```

Comment:
Maybe you could add a paragraph about interfaces, explaining that we rely more and more on them and that overridden services should implement the appropriate service rather than extending an existing class.
Although sometimes you won't have a choice but to extend it (because of type checking).

Reply:
👍